app/lib/components/term.js

```diff
@@ -52,6 +53,7 @@ export default class Term extends Component {
 
     const iframeWindow = this.getTermDocument().defaultView;
     iframeWindow.addEventListener('wheel', this.onWheel);
+    iframeWindow.addEventListener('focus', this.focus, true);
```

Comment:
this might be too agressive (ie. `this.focus`ends up running a bunch of times) but i'm not sure of how to focus the cursor otherwise, because every keystroke focuses it on the row on which you're typing